vignettes/intro.Rmd

```diff
@@ -27,6 +28,238 @@ uirender(uiinput(icon("dog"),
 ), width = "200px", height = "100px")
 ```
 
+## Create a Shiny App
+Let's create a Shiny app with `semanticPage()` and just a `h1` title.
 
-To define a new page use:
+```{r echo=TRUE, include=TRUE}
+ui <- semanticPage(
+  title = "My first page",
+  h1("My page")
+)
+
+server <- function(input, output, session) {}
+
+shinyApp(ui, server)
+```
+
+Now let's add sidebar layout `sidebar_layout()`.
```

Comment:
could you elaborate more, what is the content of sidebar_layout, that it consists of 2 components: sidebar and main, each can take a number of tags, etc.?

Reply:
done